packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container.jsx

```diff
@@ -44,7 +44,7 @@ const ProofOfIdentityContainer = ({
 
     const from_platform = getPlatformRedirect(app_routing_history);
 
-    const should_show_redirect_btn = Object.keys(platforms).includes(from_platform.ref);
+    const should_show_redirect_btn = Object.keys(platforms).includes(from_platform?.ref);
```

Comment:
Once this card goes to production, will there be any other usage of `should_show_redirect_btn`  variable and `redirect_button` button component?
If not, we must remove it and cleanup POA and POI pages, right?

Reply:
We still have Back to P2P :)